l10n_ch_qr_bill/models/res_bank.py

```diff
@@ -69,6 +81,11 @@ def _is_qr_iban(self):
         """
         return (
             self.acc_type == "iban"
-            and self._check_qr_iban_range(self.sanitized_acc_number)
-            or self.l10n_ch_qr_iban
+            and (
+                self.l10n_ch_qr_iban and
+                self.l10n_ch_qr_iban.startswith(('CH', 'LI'))
+            ) or (
```

Comment:
This should already be validated by the constraint.

```suggestion
            and self.l10n_ch_qr_iban
            or (